cloudsync/api_test.py

```diff
@@ -1,25 +1,45 @@
 """
 Tests for api
 """
-
-from io import BytesIO
+import os
+import io
 
 import boto3
 import pytest
+from botocore.exceptions import ClientError
 from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import transaction
+from django.test import override_settings
 from moto import mock_s3
 
-from cloudsync.api import process_transcode_results, refresh_status
+from cloudsync.api import process_transcode_results, refresh_status, extract_title, process_watch_file, transcode_video
```

Comment:
nit: there are enough to put them on multiple lines or do `from cloudsync import api`